### PR TITLE
[extension] Use split button for tab sharing

### PR DESCRIPTION
**Pull request by tdraier**
## Description

fixes:  https://github.com/dust-tt/tasks/issues/1643

Uses split button

Screenshot 2024-11-13 at 12 26 42

### CODE REVIEW

File: extension/app/src/components/input_bar/InputBarContainer.tsx [hunk 1]
```diff
@@ -81,6 +72,22 @@ export const InputBarContainer = ({
...
+      editorService.clearEditor();
+    });
+  };
+
+  const SendAction = {
+    label: "Send",
+    onClick,
+  };
+  const SendWithContentAction = {
+    label: "Send with tab content",
```
Comment (PopDaph) [inline extension/app/src/components/input_bar/InputBarContainer.tsx:87]: UXWriter assistant suggest: 

_Basic: "Send message"
With tab: "Send message + tab"_

WDWT?

File: extension/app/src/components/input_bar/InputBarContainer.tsx [hunk 2]
```diff
@@ -98,24 +105,6 @@ export const InputBarContainer = ({
 
       <div className="flex flex-row items-end justify-between gap-2 self-stretch pb-2 pr-2 sm:flex-col sm:border-0">
         <div className="flex py-2 space-x-1">
-          <TooltipProvider>
-            <TooltipRoot>
-              <TooltipTrigger asChild>
-                <MetaButton
```
Comment (PopDaph) [inline extension/app/src/components/input_bar/InputBarContainer.tsx:104]: ❤️

**Review by PopDaph — APPROVED** [review]
Looks awesome

**Comment by pinotalexandre** [comment]
LGTM!

File: extension/app/src/components/input_bar/InputBarContainer.tsx (hunk 1 above)
Comment (PopDaph) [reply extension/app/src/components/input_bar/InputBarContainer.tsx:87]: cc @pinotalexandre

Comment (pinotalexandre) [reply extension/app/src/components/input_bar/InputBarContainer.tsx:87]: Let's stick with the original one :)

**Pull request merged**